web-common/src/lib/event-bus/events.ts

```diff
@@ -4,15 +4,16 @@ interface Link {
 }
 
 export interface NotificationMessage {
-  type?: "default" | "success" | "error";
-  message: string;
+  type?: "default" | "success" | "loading" | "error" | "clear-all";
```

Comment:
All these types refer to different kinds of toasts. Rather than adding a `clear-all` type, I think it'd be cleaner to add a new `clear-all-notifications` event altogether.

Reply:
for sure, it could go one or another, that's why we have a review process here :)